shared/src/test/scala/com/nitro/nmesos/config/YmlSpec.scala

```diff
@@ -56,6 +55,16 @@ class YmlSpec extends Specification with YmlTestFixtures {
       modelConfig.environments("dev").singularity.slavePlacement should be equalTo (Some("SPREAD_ALL_SLAVES"))
     }
 
+    "Parse the port config from a valid Yaml file" in {
```

Comment:
The test looks good. I'd like to see a runtime validation over the yml file too (see https://github.com/Nitro/nmesos/blob/master/shared/src/main/scala/com/nitro/nmesos/config/Validations.scala)
you can run the validation with ` nmesos check`

Reply:
Done!